src/metabase/db/custom_migrations.clj

```diff
@@ -1588,15 +1588,15 @@
                                                                        handler))]
         (t2/insert! :notification_recipient (map #(merge {:notification_handler_id handler-id
                                                           :created_at              :%now
-                                                          :updated_at              :%now}%)
+                                                          :updated_at              :%now} %)
```

Comment:
would be nice to keep `%` on a separate line instead of like a dangling line ending 😁